doc/jsk_perception/nodes/craft_node.rst

```diff
@@ -0,0 +1,88 @@
+craft_node.py
+=============
+
+What is this?
+-------------
+
+.. image:: ./images/craft_node.jpg
+
+Detect text region from image.
+This is a ROS node for ``CRAFT: Character-Region Awareness For Text detection``.
+
+In order to use this feature, you need to install `pytorch <https://pytorch.org/get-started/locally/>`_ (pytorch >= 1.4.0 is recommended).
+For python2 users, download the appropriate wheel file in your environment from https://download.pytorch.org/whl/cu90/torch_stable.html (melodic) and install with pip as follow:
+
+.. code-block:: bash
+
+   pip install --user torch-1.1.0-cp27-cp27mu-linux_x86_64.whl
```

Comment:
I have a question.

You say `(pytorch >= 1.4.0 is recommended)`, but the installed torch version seems 1.1.0

Does this mean "Python3 users should use torch>=1.4.0, but Python2 users should use torch=1.1.0" ?

Reply:
OK. I modified the description.
https://github.com/jsk-ros-pkg/jsk_recognition/pull/2650/commits/a4beeaea3f6194da94404620b42d03f1024caae0
Please check it

Reply:
Thank you!
Very easy to understand.